CppSamples/DisplayInformation/BuildLegend/BuildLegend.qml

```diff
@@ -134,10 +134,10 @@ BuildLegendSample {
                             height: width
                             source: symbolUrl
                         }
-                        Text {
+                        Label {
                             anchors.verticalCenter: parent.verticalCenter
                             width: 125
-                            text: name
+                            text: qsTr(name)
```

Comment:
qsTr should only be needed when wrapping a string literal (e.g. qsTr("name")). The purpose of it is for string translations, so in this specific case, it wouldn't have any use. I suggest removing it